server/ServerFSM.cpp

```diff
@@ -609,6 +609,11 @@ sc::result MPLobby::react(const Disconnection& d) {
     // if the disconnected player wasn't in the lobby, don't need to do anything more.
     // if player is in lobby, need to remove it
     int id = player_connection->PlayerID();
+    // if player was not etablished disconnect him as AI also have player_id is Networking::INVALID_PLAYER_ID
```

Comment:
What this means isn't clear. Please elaborate.

Reply:
Changed. It's mean that non-stablished player have an id (`INVALID_PLAYER_ID`) and AI entries have same id so if it will be processed it kick AIs from lobby.

Reply:
It's mean non-established player have an id `INVALID_PLAYER_ID` so if it will be processed it kick out AIs from lobby because they have a same id.